Policies/Kubernetes/fsgroup/azurepolicy.json

```diff
@@ -0,0 +1,84 @@
+{
+  "properties": {
+    "displayName": "Control allowed FSGroups on pods in a Kubernetes Cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy controls valid ranges for FSGroup (File System Group) on volumes in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "enforceOPAConstraint",
+          "disabled"
+        ],
+        "defaultValue": "enforceOPAConstraint"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "fsgroupRule": {
```

Comment:
Can you please check if this parameter accepts case-insensitive values during policy assignment? If yes, can you check if REGO honors case-insensitive comparison?

Reply:
This parameter does not accept case-insensitive values. The Rego does not honor case insensitive comparison.

Reply:
Thanks for checking!